src/backend/distributed/commands/extension.c

```diff
@@ -77,6 +118,62 @@ ErrorIfUnstableCreateOrAlterExtensionStmt(Node *parsetree)
 }
 
 
+void
+ProcessCitusExtensionStmt(Node *parsetree)
+{
+	if (IsA(parsetree, CreateExtensionStmt))
```

Comment:
Currently this is called for every CREATE EXTENSION (e.g. postgis). It'd be nicer to do it in a UDF that is created and dropped at the start of `citus.sql`.